### ISSUE TITLE
Clicking on link in invite email leads to blank screen

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#2422]
### Expected behavior
Should receive create password screen

### Actual Behavior
Receive this dialog instead
![reaction](https://user-images.githubusercontent.com/72819/27017600-34737440-4f5c-11e7-8cdf-46304d1c9cb6.png)

### Steps to Reproduce the Behavior
1. Login as admin
1. Invite user with valid email
1. Open invitation email and visit link
1. Observe broken dialog

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.9.2
Reaction: 1.2.0
Reaction branch: development
Docker: 17.03.1-ce
```

**Comment by kieckhafer** [comment]
see #2434

**Comment by zenweasel** [comment]
Closed via #2437
